src/main/java/org/opentripplanner/middleware/controllers/api/OtpRequestProcessor.java

```diff
@@ -35,6 +33,7 @@
  */
 public class OtpRequestProcessor implements Endpoint {
     private static final Logger LOG = LoggerFactory.getLogger(OtpRequestProcessor.class);
+    private static final String USER_ID_PARAM = "userId";
```

Comment:
Not sure we need a duplicate constant for this value since it already exists on ApiController.

Reply:
Updated.